### DISCUSSION TITLE
Increase clickable area for edges

### DISCUSSION THREAD

**Discussion by Edouard-Tby** [reaviz/reagraph#243] [Ideas]
Hi there,

After engaging with Reagraph, I am impressed by its capabilities for graph visualization within React. I'd like to offer one suggestion.

**The issue is the following one :**
Using context menu on edges can be quite challenging from the user perspective (it requires to be very precise when clicking on small edges).

**Here is a potential solution :**
Create a transparent overlay shape (such as a rectangle or circle) that covers the area around the edges. This overlay will effectively increase the clickable area without visually cluttering the graph.

KR

**Comment by amcdnl** [comment]
+1 for this - we do this in reaflow already - webgl is just much trickier to do this on.

**Comment by Vincent-Bdn** [comment]
Would be really valuable for big graphs especially indeed. Looking forward to this feature if you decide to implement it. Thanks
